data/en/test_date_validation_mm_yyyy_combined.json

```diff
@@ -0,0 +1,77 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "023",
+    "title": "Date formats",
+    "description": "A test schema for different date formats",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "variables": {
```

Comment:
@dwyeradam this can be removed as part of the metadata work later on

Reply:
Done